sdk/storage/Azure.Storage.Blobs/tests/Azure.Storage.Blobs.Tests.csproj

```diff
@@ -23,4 +23,9 @@
     </None>
     <None Include="$(AzureStorageSharedTestSources)\azurite_cert.pem" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
+    <ItemGroup> 
+    <Content Include="Resources\**"> 
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory> 
+    </Content> 
+  </ItemGroup>
```

Comment:
Just curious what does this do?

Reply:
This says to copy the "Resources" directory with the build.  I have a parquet file that we upload as part of the Quick Query Parquet test, I put the file in Azure.Storage.Blobs.Tests/Resources.  Similar to the test files we have in Azure.StorageInternal.Avro.Tests/Resources.